cordova-lib/src/plugman/uninstall.js

```diff
@@ -145,10 +145,9 @@ module.exports.uninstallPlugin = function(id, plugins_dir, options) {
         var deps = pluginInfo.getDependencies();
         var deps_path;
         deps.forEach(function (d) {
-            var splitVersion = d.id.split('@');
-            deps_path = path.join(plugin_dir, '..', splitVersion[0]);
+            deps_path = path.join(plugin_dir, '..', cordovaUtil.extractPluginId(d.id));
             if (!fs.existsSync(deps_path)) {
-                var newId = pluginMapper[splitVersion[0]];
+                var newId = pluginMapper[cordovaUtil.parseRegistryPluginSpec(d.id)[0] || d.id];
```

Comment:
You use `extractPluginId` in L148 but `parseRegistryPluginSpec` here but they will return different results in case if there is a scoped dependency in plugin.xml. Is that intentional?